api/models/LinkedAccount.js

```diff
@@ -61,7 +61,7 @@ module.exports = bookshelf.Model.extend({
       case 'facebook':
         return {
           facebook_url: profile.profileUrl || get(profile, '_json.link'),
-          avatar_url: `https://graph.facebook.com/${profile.id}/picture?type=large`
+          avatar_url: `https://graph.facebook.com/${profile.id}/picture?type=large&access_token=${process.env.FACEBOOK_APP_ID}|${process.env.FACEBOOK_CLIENT_TOKEN}`
```

Comment:
I had already added our `FACEBOOK_CLIENT_TOKEN` to `hylo-node`, otherwise I'm not understanding your question. The `access_token` IS a combination of the app id and client token (`access_token={APP_ID}|{CLIENT_TOKEN}')